push_notifications/conf/app.py

```diff
@@ -26,7 +26,8 @@
 	"APNS",
 	"FCM",
 	"GCM",
-	"WNS"
+	"WNS",
+	"BROWSER"
```

Comment:
Why use the key **BROWSER** instead of **WEB** or **WEBP**?

Reply:
What about WPNS (Web push notification service)?

Reply:
To better clarify my position. I think BROWSER is a little to generic. You can push to a browser (Google Chrome) using just FCM/GCM as well. @jamaalscarlett's suggestion is also good. Just not **BROWSER**.

Reply:
WPNS is very similar to WNS. Also the protocol name is WebPush. If you want I or you can change it for WP, but WP is Windows Phone acronym. 

I had doubts about this name

Reply:
I'm fine with WP

Reply:
Go with **WP**. Windows phone has been EOLd so I do not think that will be a concern.

Reply:
Done [19ac0c1e29402aee6a436726a3e892a08f2bc07f]

Reply:
Now done [facad6cfc696a83c9667f194450b296c9e1f397f]